dds_cli/text_handler.py

```diff
@@ -65,13 +66,18 @@ def task_name(file, step="", max_len=30):
                 task_name = "..." + task_name.split("...", 1)[-1][-max_len::]
 
         # Different symbols to the left depending on current process
-        symbol = ""
-        if step == "encrypt":
-            symbol = ":lock:"
-        elif step == "put":
-            symbol = ":arrow_up: "
-        elif step == "get":
-            symbol = ":arrow_down: "
-        elif step == "decrypt":
-            symbol = ":unlock:"
-        return f"{symbol} {task_name}"
+        if dds_cli.dds_on_legacy_console:
+            symbols = {
+                "encrypt": "[bold]seal",
```

Comment:
Small thing here but I think it's better to have `encrypt` and `decrypt` instead of `seal` and `open`, respectively

Reply:
So you have no concerns regarding long status codes of variable length? Just asking, because I believed it might affect the columnar arrangement and ultimately readability when working with many files. I noticed you specifically created a function to truncate long filenames and believed it was for this reason...